specification/cognitiveservices/data-plane/Language/readme.md

```diff
@@ -6,22 +6,12 @@ This is the AutoRest configuration file the Cognitive Services Language SDK.
 
 ## Releases
 
-The current preview release is 2022-02-01-preview
+The current preview release of Conversational Language Understanding and Text Analytics is 2022-03-01-preview.
 
-The current stable release of QuestionAnswering is 2021-10-01 and preview release of Luis Deepstack is 2021-07-15-preview
+The current stable release of Question Answering is 2021-10-01.
 
 ```yaml
-tag: release_2022_02_01_preview
-add-credentials: true
-clear-output-folder: true
-openapi-type: data-plane
-directive:
-  - suppress: LongRunningResponseStatusCode
-    reason: The validation tools do not properly recognize 202 as a supported response code.
-```
-
-```yaml
-tag: release_2021_11_01_preview
+tag: release_2022_03_01_preview
```

Comment:
This should be updated for the April release i.e. `release_2022-04-01_preview` and then add a block for that as well, basically copying lines 23 through 35 (in the right/new file) below.

Each package/release shoul dhave it's own yaml section, but the top-level `tag:` and friends are updated for each subsequent release. Before they were duplicated, and it wasn't clear from diffs until I merged March's readme.md.

Reply:
Also, since you're removing analyzeconversations.json, make sure the release_2022-04-01-preview section doesn't reference it in `input-file:`.